app/views/new-overview.html

```diff
@@ -365,6 +365,7 @@ <h2 ng-if="overview.state.serviceInstances">
                   <service-instance-row
                     ng-repeat="serviceInstance in overview.state.orderedServiceInstances"
                     api-object="serviceInstance"
+                    bindings="overview.state.bindingsByInstanceRef[serviceInstance.metadata.name]"
```

Comment:
Generally the items in `state` are meant to be accessed directly by the row. So maybe `bindingsByInstanceRef` should just be on `overview` directly.

But I'm OK either way.